### PR TITLE
82 android deployment

### LINKED ISSUE #82: Android Deployment
Add documentation for:
- [ ]  Cutting an APK for client testing
- [ ] Setting up a Developer account for Google Play
- [ ] Deploying to Google Play Store for production
- [ ] Setting up Android Studio with physical devices and virtual devices

### PR DESCRIPTION
**Pull request by j-ornelas**
closes #82 

Adds docs explaining some basic development / deployment processes for Android, specifically catered to developers working with React Native.

### CODE REVIEW

File: native/react-native/android.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	- Google Play offeres a series of questions that will automatically provide a content rating for you
+
+### Creating an APK
+
+#### What is an APK?
+The APK is the file that represents your application. It is uploaded to the Google Play Store and is eventually downloaded on the user's phone, letting them run your application. Emulators and dev-mode phones can run unsigned APKs, but you'll need to create a signed APK to distrube your app to users.
+
+#### What is a signed APK?
+A signed APK is an APK that has an attached keystore file. The keystore file contains some additional information about the app and the app uploader. If you're putting the app on the Play Store for the first time, you'll need to create a keystore file. If you're updating an existing app, you'll need the existing keystore before you can update the app on the Google Play Store. **IMPORTANT NOTE:**  Make sure you backup your Keystore after creating it. Once an app is published to the Play Store, it will be impossible to update it if you lose the keystore file or password.
+
```
[the remark this answers is not in the document]
Comment (coreyshuman) [reply native/react-native/android.md]: We will be implementing a password manager and password/key storage requirements at Shift3 soon. I'm test driving a few password managers for our purposes right now.

[the remark this answers is not in the document]
Comment (ryekerjh) [reply native/react-native/android.md]: @coreyshuman Might I recommend Myki?

**Comment by ryekerjh** [comment]
@j-ornelas Would you mind pulling in from master and moving this file to `native/react-native/deployment/android/README.md`?

**Comment by j-ornelas** [comment]
@ryekerjh will do!

**Pull request merged**